clang/test/dpct/cmake_migation/case_013/case_013_cuda_compile_cubin.cpp

```diff
@@ -0,0 +1,10 @@
+// RUN: rm -rf %T && mkdir -p %T
+// RUN: cd %T
+// RUN: cp %S/input.cmake ./input.cmake
+// RUN: dpct -in-root ./ -out-root out  ./input.cmake --rule-file=%S/../../../../tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml --migrate-cmake-script-only --cuda-include-path="%cuda-path/include"
```

Comment:
file=%S/../../../../tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml and  --cuda-include-path="%cuda-path/include" could be removed, just like https://github.com/oneapi-src/SYCLomatic/blob/ba2e8ac690a5a66fac50027935709b2491900fbf/clang/test/dpct/cmake_migation/case_001/case_001_framework.cpp#L4